Documentation/ApiOverview/CommandControllers/Index.rst

```diff
@@ -46,6 +46,7 @@ Creating a new Command in Extensions
 
        Vendor\Extension\:
          resource: '../Classes/*'
+         exclude: '../Classes/Domain/Model/*'
```

Comment:
This should be moved to the general DI description instead:
https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/DependencyInjection/Index.html

Also we should add an explanation about this on that page.

Finally we could reduce this code snippet to the command and link to that page.